### PR TITLE
BUGZ-1081: Put Macs into "Low" rendering tier if INtel GPU

### PR DESCRIPTION
**Pull request by samcake**
https://highfidelity.atlassian.net/browse/BUGZ-1081

TEST PLAN:
on fresh install on a Mac,we should see that the recommended profile is:
Tier LOW if the gpu is Intel
Tier MID else

The current master behavior was to assign  mid for macbook pro (regardless of the gpu) and low for the other types of mac computers

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20861/HighFidelity-Beta-PR16060-eb24667.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/20861/HighFidelity-Quest-Beta-PR16060-eb24667.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20861/HighFidelity-Beta-PR16060-eb24667.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/20861/HighFidelity-Beta-Interface-PR16060-eb24667.dmg

File: libraries/platform/src/platform/Profiler.cpp [hunk 1]
```diff
@@ -58,21 +58,31 @@ bool filterOnComputer(const platform::json& computer, Profiler::Tier& tier) {
...
             tier = Profiler::Tier::MID;
             return true;
-        } else if (model.find("MacBook") != std::string::npos) {
-            tier = Profiler::Tier::LOW;
-            return true;
         }
+    } else {
+        // no gpuinfo ?
+        // Go low in doubt
+        // go LOW because Intel GPU
```
Comment (AndrewMeadows) [inline libraries/platform/src/platform/Profiler.cpp]: `// Go low in doubt` yes
`// go LOW because Intel GPU` no (this looks like a copy-n-paste error)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20861/HighFidelity-Beta-PR16060-eb24667.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/20861/HighFidelity-Beta-Interface-PR16060-eb24667.exe

**Pull request merged**